### ISSUE TITLE
Náhrada grafémů: odděl od ruční segmentace

### ISSUE THREAD

**Issue by Glutexo** [pelegrinova/morfo_segmentace#28]
~~A doplň sem popis.~~

Doplněno:

Náhrada grafémů za fonémy je činnost nesouvisející se samotnou morfologickou segmentací. Jedná se o oddělený proces, jenž si zaslouží vlastní skript.

[Manuální segmentace](https://github.com/pelegrinova/morfo_segmentace/blob/9fd5717345395329a163d0d2f57892472b3cb5e8/morfo_segmentace_manuáln%C3%AD.py) v tuto chvíli řeší obojí. Oddělením se zvýší symetrie mezi manuální segmentací, jež v tuto chvíli dělá i substituci, a automatickou, jež očekává již přepsaný text.

Až se skripty naučí pracovat se vstupy a výstupy standardním způsobem, bude možné tyto dva kroky snadno zřetězit.

**Comment by pelegrinova** [comment]
Prosím o psaní "fonetický" - s uvozovkami. Nejedná se ani náhodou o fonetický přepis, jen mu pro sebe pracovně říkám "fonetický". Spíš je to substituce grafémů za fonémy, ale to bylo moc dlouhé...

**Comment by pelegrinova** [comment]
Myslíš tím do samostatného pythonního souboru?

**Comment by Glutexo** [comment]
Ano. Popíšu podrobněji. Motivaci i provedení. Založil jsem jen rychle, ať na to nezapomenu.

Můžeme mu říkat třeba pseudofonetický?

**Comment by pelegrinova** [comment]
Asi ano. Byť spokojenější bych byla s "substituce grafémů", ale "pseudofonetický" se zdá být praktičtější pro pojmenovávání proměnných i...
Ok, těším se na detaily, i když mám pocit, že tuším. Minimálně motivaci rozumím. Ale možná ne tak detailně, tož to tu klidně popis pořádně.

**Comment by Glutexo** [comment]
Doplněno. Zas takové dílo to nakonec nebylo.

**Comment by Glutexo** [comment]
Do záznamu: #30 se nakonec zřejmě slučovat nebude. Ponecháváme zatím pro případnou referenci.
